### DISCUSSION TITLE
能否支持隐藏单张sim卡

### DISCUSSION THREAD

**Discussion by Dagwbl** [MonwF/customiuizer#45] [Ideas]
如题，双排导航栏有些太宽了，感觉整体不大协调，想隐藏一张卡。

另外小米10，MIUI13稳定版，软重启无效

感谢作者的开发

**Comment by MonwF** [comment]
隐藏sim卡后期可能会添加
软重启那个没环境 没办法解决

**Reply by Dagwbl** [reply]
嗯嗯，好的，感谢。
软重启不影响软件主体功能没事。

**Reply by MonwF** [reply]
> 嗯嗯，好的，感谢。 软重启不影响软件主体功能没事。

双排信号图标太宽吗？隐藏sim卡其他很多模块支持

**Reply by Dagwbl** [reply]
> > 嗯嗯，好的，感谢。 软重启不影响软件主体功能没事。
> 
> 双排信号图标太宽吗？隐藏sim卡其他很多模块支持

可以参考一下这个状态栏，旁边的4G和这个大小差距有些大。
![Screenshot_2022-10-21-22-56-24-793_com.tencent.mm.jpg](https://user-images.githubusercontent.com/39078825/197226065-1de1f04d-226c-4ea8-837c-494476e911af.jpg)

**Comment by MonwF** [comment]
双排信号栏 4g 字体大小之前在缩放情况下确实不合理，调整了下。。。
图标太宽的问题，没空制作ios那样的4格图标

**Comment by MonwF** [comment]
已经添加隐藏单张sim卡功能 @Dagwbl

**Reply by Dagwbl** [reply]
太强了！那个图标或许我可以帮忙制作，就是不大清楚图标设计规范

**Reply by Dagwbl** [reply]
> 已经添加隐藏单张sim卡功能 @Dagwbl

才发现更新说明里面已经重新设计过了
